src/workspacer.FSharpConfig/FSharpCompileException.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace workspacer.FSharpConfig
+{
+    using System;
+    using FSharp.Compiler.Diagnostics;
+
+    public class FSharpCompileException : Exception
+    {
+        public FSharpDiagnostic[] Errors { get; }
+
+        public FSharpCompileException(string message, FSharpDiagnostic[] diagnostics) : base(message)
```

Comment:
If the property is called `Errors`, it would be more consistent to name the parameter `errors` as well.